lib/react_on_rails/version_checker.rb

```diff
@@ -74,20 +77,24 @@ def self.build
       end
 
       def self.package_json_path
-        Rails.root.join("client", "package.json")
+        Rails.root.join(ReactOnRails.configuration.node_modules_location, "package.json")
```

Comment:
could `ReactOnRails.configuration.node_modules_location` be nil and if so, what would happened?

Reply:
The default is an empty string, so it would have to be manually configured to `nil`.

If it is `nil`, that will cause a fatal error.